packages/react/src/table/TableScroller.tsx

```diff
@@ -23,6 +32,8 @@ export function TableScroller({ children }: TableScrollerProps) {
 	const [thumbPosition, setThumbPosition] = useState(0);
 	const [thumbWidthRatio, setThumbWidthRatio] = useState(0);
```

Comment:
Not your doing, but just noticed lots of flash of scrollbars disappearing on mount, so we should assume that it doesn't have scrollbars until it does
```suggestion
	const [thumbWidthRatio, setThumbWidthRatio] = useState(1);
```

Reply:
Yeah ok, I'll look into this. Knowing not everyone has the `TableWrapper`, I think we'll need a different approach to this scrollbar + overlay with frozen columns functionality.

Reply:
I was thinking that scrollbar should still just be in TableWrapper, so that way, people who use it get the benefits of the scrollbar and those that don't, don't, but they may have their own solution or none is needed.

The other stuff should be moved out though.

If this isn't technically possible, then we'll have to revert TableWrapper to what it used to be and move the scroller elsehwere

Reply:
Yeah, the scrollbar needs the overlay according to our designs. The overlay doesn't need x-axis offsets if there's no frozen columns. So, I think it'll be all fine to do that part. If we can put the scroller inside the `Table`, then we don't need the extra context I added. Just need to look into it more.